lldb/source/Plugins/ScriptInterpreter/Python/ScriptInterpreterPython.cpp

```diff
@@ -2037,19 +2037,28 @@ lldb::ValueObjectSP ScriptInterpreterPythonImpl::GetChildAtIndex(
   return ret_val;
 }
 
-int ScriptInterpreterPythonImpl::GetIndexOfChildWithName(
+llvm::Expected<int> ScriptInterpreterPythonImpl::GetIndexOfChildWithName(
     const StructuredData::ObjectSP &implementor_sp, const char *child_name) {
   if (!implementor_sp)
-    return UINT32_MAX;
+    return llvm::createStringError(
+        "'SyntheticChildrenFrontEnd::ScriptInterpreterPythonImpl' cannot find "
```

Comment:
```suggestion
        "'ScriptInterpreterPythonImpl' cannot find "
```
(applies to other places in this function too)